### ISSUE TITLE
Add an indicator that a message is being loaded

### ISSUE THREAD

**Issue by tadzik** [runbox/runbox7#611]
This is mostly visible when toggling the HTML message, but not exclusively. Plaintext messages are pre-fetched when they're visible on the screen, but you can observe a delay if you scroll to the bottom of a big folder and click on a random message.

There is a noticeable delay between opening the message panel and any content actually showing in cases where additional network requests need to be performed. In case of HTML messages one can see a "This HTML message has been sanitized for your security" note but nothing else, making it seems like there's nothing more coming. In case of plaintext messages there is just a big white space, which ironically somewhat indicates that something is not working *yet* ;)

Some indication that the content is coming would be nice.

**Comment by gtandersen** [comment]
Agreed @tadzik -- there could also be ways to make HTML content load faster.

**Comment by gtandersen** [comment]
Ideally we don't need an indicator at all as message content should load immediately except for cases where there is no Internet connection.

In the case of heavy HTML messages we should load the textual content first and then any media.
